### PR TITLE
Update bugs-found.md

### PR DESCRIPTION
**Pull request by Uclydde**
1. Add the corresponding patches to the existing commits.
2. Fix the dates on the existing commits.
2. Consolidate and remove duplicates.
3. Add one more recent commit.

### CODE REVIEW

**Comment by necipfazil** [comment]
The commit link for `mips: fix unmet dependency for FRAME_POINTER when !ARCH_WANT_FRAME_POINTERS` seem to belong to a different fix.

Likewise, the patch link for `mips: fix unmet dependency for DEBUG_INFO when !DEBUG_KERNEL` seem to belong to a different one.

Could you fix these, and double-check the rest of the newly added ones?

Thanks!

**Comment by Uclydde** [comment]
> The commit link for `mips: fix unmet dependency for FRAME_POINTER when !ARCH_WANT_FRAME_POINTERS` seem to belong to a different fix.
> Likewise, the patch link for `mips: fix unmet dependency for DEBUG_INFO when !DEBUG_KERNEL` seem to belong to a different one.

Good catch - these are fixed now.
> Could you fix these, and double-check the rest of the newly added ones?

Yep, there was a missing patch link as well. I added that in, too.

File: docs/bugs_found.md [hunk 1]
```diff
@@ -66,45 +73,53 @@
...
 
-2020-04-09 [Commit](https://cgit.freedesktop.org/drm/drm-misc/commit/?id=62066d3164467167fc27b2383f67d097e39bf176): Committed into drm-misc-next.
+2021-04-01 [Commit](https://cgit.freedesktop.org/drm/drm-misc/commit/?id=62066d3164467167fc27b2383f67d097e39bf176): Committed into drm-misc-next.
+
+2021-02-22 [Patch](https://lkml.org/lkml/2021/2/22/1167)
 
 ### mips: fix unmet dependency for FRAME_POINTER when !ARCH_WANT_FRAME_POINTERS
 
-2020-04-09 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=f766b28a5edfd86600e55360cc4bf29c71cca2eb): Committed into linux-next.
+2021-03-29 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=7d37cb2c912dc5c25ffac784a4f9b98c06c6bd08): Committed into linux-next.
```
Comment (necipfazil) [inline docs/bugs_found.md:90]:
```suggestion
2021-04-09 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=7d37cb2c912dc5c25ffac784a4f9b98c06c6bd08): Committed into linux-next.
```

File: docs/bugs_found.md [hunk 2]
```diff
@@ -66,45 +73,53 @@
...
-### staging: rtl8192e: fix unmet dependency for CRYPTO_MICHAEL_MIC and CRYPTO_LIB_ARC4 when !CRYPTO
+2021-03-10 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=196ebe5c000afbfe67b8561f716e365174552bd7): Committed into linux-next.
 
-2020-04-09 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=7c36194558cf49a86a53b5f60db8046c5e3013ae): Committed into linux stable.
+2021-02-24 [Patch](https://lkml.org/lkml/2021/2/24/1184)
 
 ### fs: nfsd: fix unmet dependency for CRYPTO_SHA256 and CRYPTO_MD5 when !CRYPTO
 
-2020-04-09 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=7005227369079963d25fb2d5d736d0feb2c44cf6): Committed into linux stable.
+2021-03-06 [Commit](https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=7005227369079963d25fb2d5d736d0feb2c44cf6): Committed into linux stable.
```
Comment (necipfazil) [inline docs/bugs_found.md:120]: Looks like a commit to linux-next (not linux stable).

This should be ` Committed into linux-next`, or the link should be to the stable tree if it's really the case.

**Review by necipfazil — CHANGES_REQUESTED** [review]
Thanks for the fixes.

Seems like there are 2 more nits. Could you fix them?

Besides, LGTM

File: docs/bugs_found.md (hunk 1 above)
Comment (Uclydde) [reply docs/bugs_found.md]: Okay, I fixed those, and then added links for any linux-stable commits as well.

**Review by necipfazil — APPROVED** [review]

**Review by paulgazz — APPROVED** [review]

**Pull request merged**